package/rpi-firmware/rpi-firmware.mk

```diff
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-RPI_FIRMWARE_VERSION = 66bafab005569e3eb92ec54cd3efeee3da338738
+RPI_FIRMWARE_VERSION = 67bfaeb8457737053d8bab414d3d21c7587043cf
```

Comment:
I prefer to always go with their latest "stable" version, since it's the one used by the latest Raspbian, unless there's a very specific need that we use a different version. I always take the stable commit from their [stable branch](https://github.com/raspberrypi/firmware/tree/stable). At the time of writing, that would be `e10b2880e26708afc0b32a485c1231207037fc45`.

The same applies to Linux Kernel commit. They use the `extra/git_hash` file to mark which git commit they use for a certain firmware version. At the time of writing, you can find it [here](https://github.com/raspberrypi/firmware/blob/e10b2880e26708afc0b32a485c1231207037fc45/extra/git_hash).

Reply:
That makes perfect sense. Better than always jumping to try the latest. Thanks very much for the links! I was not aware of those commits.